spacy/lang/nl/lemmatizer/_adjectives.py

```diff
@@ -0,0 +1,4584 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+
+ADJECTIVES = set(
```

Comment:
This list seems to contain quite a lot of verbs that I can't imagine being adjectives, like `klussen` or `verwilderen` ?

Reply:
Going to pitch in here so as to keep momentum high. (I worked together with Yves on this pull request.)

That's an interesting point about sorting the strings. Looks like that's what the other languages with lemmatization data do. Beyond conforming to that convention, however, there isn't much to be gained from it. It's not recommended to edit the lemmatization data manually (neither the lemma indexes "_<pos>.py" nor the exceptions indexes in "_<pos>_irreg.py" ), since the data is in fact extracted entirely automatically (even the serialization/persistence as a python module happens automatically). So any improvements to the data should focus on the data extraction pipeline (which currently resides in an ugly Jupyter notebook, but will be shared once it's cleaned up).

That brings us to the second observation. I'll run some tests to see how those cases slipped in. In any case, it looks like they're also included correctly in "_verbs.py", so the impact on lemmatization accuracy should be minimal.